packages/client/src/resources/guardian/users.ts

```diff
@@ -1,17 +1,17 @@
-import { ApiResponse, ApiResponseWithPagination, Resource } from "@arkecosystem/client";
+import { ApiQuery, ApiResponse, ApiResponseWithPagination, Resource } from "@arkecosystem/client";
 
-import { User } from "../../resources-types/guardian";
+import { User, UserGroups } from "../../resources-types/guardian";
 
 export class Users extends Resource {
-    public async index(): Promise<ApiResponseWithPagination<User>> {
+    public async index(query?: ApiQuery): Promise<ApiResponseWithPagination<User>> {
         return this.sendGet("guardian/users");
     }
 
     public async get(publicKey: string): Promise<ApiResponse<User>> {
         return this.sendGet(`guardian/users/${publicKey}`);
     }
 
-    public async userGroups(publicKey: string): Promise<ApiResponse<User>> {
+    public async userGroups(publicKey: string): Promise<ApiResponse<UserGroups>> {
```

Comment:
Can we remove `UserGroups` interface and replace it with existing `Group` interface?